lms/templates/register.html

```diff
@@ -16,6 +16,15 @@
 <%! from third_party_auth import pipeline, provider %>
 <%! import calendar %>
 
+<%
+  if self.theme_enabled() and not self.is_microsite():
```

Comment:
We use this same `if` statement throughout, could it be simplified into a single method call, something like

```python
if self.is_non_microsite_theme():
   pass
```